Resources/Textures/LobbyScreens/attributions.yml

```diff
@@ -0,0 +1,9 @@
+- files: ["supermatter.png", "robotics.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Veritius#2351 on discord"
```

Comment:
Can you do `Veritius#2351 (257233913951289344) on discord / @Veritius on GitHub`